From b7ecb4d6ad393da862831c156f833925353b8a14 Mon Sep 17 00:00:00 2001
From: Giuseppe Scrivano <gscrivan@redhat.com>
Date: Mon, 9 Dec 2013 14:19:16 +0100
Subject: [PATCH 05/10] osdict: append "and later" to the last version of an OS

It makes clearer to the user to pick the last version available for an
OS when a newer version is not listed by virt-manager/virt-install.

Closes: https://bugzilla.redhat.com/show_bug.cgi?id=1011290
Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>
(cherry picked from commit 25723f98ae35fa594720449023dba9cf10a25442)

Conflicts:

	virtinst/osdict.py
---
 virtinst/osdict.py | 26 +++++++++++++-------------
 1 file changed, 13 insertions(+), 13 deletions(-)

diff --git a/virtinst/osdict.py b/virtinst/osdict.py
index 728afeb..50e6dfb 100644
--- a/virtinst/osdict.py
+++ b/virtinst/osdict.py
@@ -268,7 +268,7 @@ OS_TYPES = {
         }
     },
     "rhel7": {
-        "label": "Red Hat Enterprise Linux 7",
+        "label": "Red Hat Enterprise Linux 7 and later",
         "distro": "rhel",
         "supported": False,
         "devices" : {
@@ -383,7 +383,7 @@ OS_TYPES = {
         }
     },
     "fedora18": {
-        "label": "Fedora 18",
+        "label": "Fedora 18 and later",
         "distro": "fedora",
         "supported": True,
         "devices" : {
@@ -403,7 +403,7 @@ OS_TYPES = {
         },
     },
     "opensuse12": {
-        "label": "openSuse 12",
+        "label": "openSuse 12 and later",
         "distro": "suse",
         "supported": True,
         "devices" : {
@@ -418,7 +418,7 @@ OS_TYPES = {
         "supported": True,
     },
     "sles11": {
-        "label": "Suse Linux Enterprise Server 11",
+        "label": "Suse Linux Enterprise Server 11 and later",
         "distro": "suse",
         "supported": True,
         "devices" : {
@@ -493,7 +493,7 @@ OS_TYPES = {
         }
     },
     "debianwheezy": {
-        "label": "Debian Wheezy",
+        "label": "Debian Wheezy and later",
         "distro": "debian",
         "sortby": "debian7",
         "supported": True,
@@ -580,7 +580,7 @@ OS_TYPES = {
         },
     },
     "ubuntuquantal": {
-        "label": "Ubuntu 12.10 (Quantal Quetzal)",
+        "label": "Ubuntu 12.10 (Quantal Quetzal) and later",
         "distro": "ubuntu",
         "supported": True,
         "devices" : {
@@ -646,7 +646,7 @@ OS_TYPES = {
         "distro": "winserv",
     },
     "win2k8": {
-        "label": "Microsoft Windows Server 2008",
+        "label": "Microsoft Windows Server 2008 and later",
         "supported": True,
         "sortby": "mswinserv2008",
         "distro": "winserv",
@@ -658,7 +658,7 @@ OS_TYPES = {
         "distro": "win",
     },
     "win7": {
-        "label": "Microsoft Windows 7",
+        "label": "Microsoft Windows 7 and later",
         "supported": True,
         "sortby": "mswin7",
         "distro": "win",
@@ -677,13 +677,13 @@ OS_TYPES = {
         "label": "Sun Solaris 9",
     },
     "solaris10": {
-        "label": "Sun Solaris 10",
+        "label": "Sun Solaris 10 and later",
         "devices" : {
             INPUT : USB_TABLET,
         },
     },
     "opensolaris": {
-        "label": "Sun OpenSolaris",
+        "label": "Sun OpenSolaris and later",
         "devices" : {
             INPUT : USB_TABLET,
         },
@@ -710,7 +710,7 @@ OS_TYPES = {
         },
     },
     "freebsd8": {
-        "label": "FreeBSD 8.x" ,
+        "label": "FreeBSD 8.x and later" ,
         "supported": True,
         "devices" : {
             NET : { "model" : [ (HV_ALL, "e1000") ] }
@@ -718,7 +718,7 @@ OS_TYPES = {
     },
 
     "openbsd4": {
-        "label": "OpenBSD 4.x" ,
+        "label": "OpenBSD 4.x and later" ,
         # http://calamari.reverse-dns.net:980/cgi-bin/moin.cgi/OpenbsdOnQemu
         # https://www.redhat.com/archives/et-mgmt-tools/2008-June/msg00018.html
         "devices" : {
@@ -746,7 +746,7 @@ OS_TYPES = {
         "label": "Novell Netware 5",
     },
     "netware6": {
-        "label": "Novell Netware 6",
+        "label": "Novell Netware 6 and later",
         "pv_cdrom_install": True,
     },
 
-- 
1.8.5.3

